lib/languages.js

```diff
@@ -94,6 +94,12 @@ export const languages = {
         extensions: ['.cl', '.ocl'],
         alias: [],
     },
+    cpp_for_opencl: {
+        name: 'C++ for OpenCL',
```

Comment:
So, we already have the language "OpenCL C". Ideally I'd like them named similarly, so either both "OpenCL C" and "OpenCL C++" or "C for OpenCL" and "C++ for OpenCL" . Which is the more correct?

Reply:
So there is a language called "OpenCL C++" which is different from "C++ for OpenCL". 
OpenCL C++ has now been deprecated and has been replaced by C++ for OpenCL.

The correct names would be OpenCL C and C++ for OpenCL... which unfortunately doesn't really fit in with trying to name them similarly :(

Reply:
I agree that consistent naming would be great but for various reasons unfortunately we didn't end with the consistent naming in the OpenCL community. :(

OpenCL C++ is a language that has been published but has not been implemented/used in the community. So it is better to avoid using its name otherwise we might create confusion. For example, in clang we use C++ for OpenCL name:
https://clang.llvm.org/docs/UsersManual.html#c-for-opencl

Also OpenCL C is the official name of the C99-based variant of OpenCL kernel language. Here are more details about the OpenCL kernel languages:
https://www.khronos.org/opencl/
https://github.com/KhronosGroup/OpenCL-Guide/blob/main/chapters/cpp_for_opencl.md

Reply:
Understood - thanks for clarifying. This looks good to me then!